includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -1055,6 +1056,20 @@ public function maybe_filter_gateway_title( $title, $id ) {
 		return $title;
 	}
 
+	/**
+	 * Sets the payment method title on the order for emails.
+	 *
+	 * @param WC_Order $order   WC Order object.
+	 * @param bool     $sent_to_admin Whether the email is being sent to the admin or not.
+	 * @param bool     $plain_text Whether the email is being sent as plain text or not.
+	 */
+	public function set_payment_method_title_for_email( $order, $sent_to_admin, $plain_text ) {
```

Comment:
Is there any reason to keep the two last unused parameters in this method definition, or can we go with:
```suggestion
	public function set_payment_method_title_for_email( $order ) {
```
?

Reply:
I thought so, but now that you've mentioned it, I believe these unused variables can be removed. Took care of it in [cba6d9e](https://github.com/Automattic/woocommerce-payments/pull/6351/commits/cba6d9e79464e36efa32d24d7069194121e9922c).